### PR TITLE
Remove web-console and spring from test group on default Gemfile.

### PR DESCRIPTION
**Pull request by kassio**

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/templates/Gemfile [hunk 1]
```diff
@@ -25,7 +25,11 @@ group :development, :test do
 <% if RUBY_ENGINE == 'ruby' -%>
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug'
+<% end -%>
+end
 
+group :development do
```
Comment (rafaelfranca) [inline railties/lib/rails/generators/rails/app/templates/Gemfile]: Should not this be inside the `if`?

**Comment by rafaelfranca** [comment]
This is also following https://github.com/rails/web-console/pull/134 cc @chancancode

File: railties/lib/rails/generators/rails/app/templates/Gemfile (hunk 1 above)
Comment (kassio) [inline railties/lib/rails/generators/rails/app/templates/Gemfile]: Yeah, you're right. Fixing.

**Comment by rafaelfranca** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by gsamokovarov** [comment]
Thank you @kassio!

**Comment by cbeer** [comment]
Can/should this be backported to 4.2? Now that web-console 2.2.0 is released, the default rails-provided Gemfile is broken.

**Comment by rafaelfranca** [comment]
@cbeer done 121baf2

**Comment by cbeer** [comment]
:clap:

**Comment by gsamokovarov** [comment]
I've release web-console 2.2.1, which changes keeps this off by default, so I don't keep the tests on newly created rails apps broken. I'll flip it back when 4.2.4 is released. Thank you for the quick reports!

**Pull request merged**